src/components/ItemRecognition/ItemRecognition.js

```diff
@@ -10,7 +10,7 @@ import MobileNet from '../Admin/Trainer/MobileNet';
 import './ItemRecognition.css';
 
 const TIMEOUT_IN_SECONDS = 10;
-const ML_THRESHOLD = 0.35;
+const ML_CERTAINTY = 3;
```

Comment:
Is there any way we could make this easier to know exactly what it's used for at a glance?

It's almost how many of the same guesses do we need to be certain

Not sure what's the best for this. Might be good to ask another member of the team, explain what it's used for and that'll show us a good descriptive name for it

Reply:
MIN_CONSECUTIVE_GUESSES/MATCHES or something?